From: "XueBing Chen" <chenxuebing@jari.cn>
Subject: mm: sparsemem: drop unexpected word 'a' in comments
Date: Sat, 25 Jun 2022 16:51:35 +0800 (GMT+08:00)

there is an unexpected word 'a' in the comments that need to be dropped

Link: https://lkml.kernel.org/r/24fbdae3.c86.1819a0f31b9.Coremail.chenxuebing@jari.cn
Signed-off-by: XueBing Chen <chenxuebing@jari.cn>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/sparse-vmemmap.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/sparse-vmemmap.c~mm-sparsemem-drop-unexpected-word-a-in-comments
+++ a/mm/sparse-vmemmap.c
@@ -556,7 +556,7 @@ pte_t * __meminit vmemmap_pte_populate(p
 		} else {
 			/*
 			 * When a PTE/PMD entry is freed from the init_mm
-			 * there's a a free_pages() call to this page allocated
+			 * there's a free_pages() call to this page allocated
 			 * above. Thus this get_page() is paired with the
 			 * put_page_testzero() on the freeing path.
 			 * This can only called by certain ZONE_DEVICE path,
_
